containers/daap-presigned-url/tests/unit/mock_test.py

```diff
@@ -11,6 +12,7 @@ def test_success(s3_client, fake_context, region_name, monkeypatch):
     bucket_name = "bucket"
```

Comment:
💡 Suggestion: Can we pull these constants out into a dict we're passing to mark.parameterize?